consensus/polybft/state_store_epoch.go

```diff
@@ -154,17 +171,19 @@ func (s *EpochStore) insertEpoch(epoch uint64, dbTx *bolt.Tx) error {
 }
 
 // isEpochInserted checks if given epoch is present in db
-func (s *EpochStore) isEpochInserted(epoch uint64) bool {
+func (s *EpochStore) isEpochInserted(epoch uint64, chainID uint64) bool {
 	return s.db.View(func(tx *bolt.Tx) error {
-		_, err := getEpochBucket(tx, epoch)
+		_, err := getEpochBucket(tx, epoch, chainID)
 
 		return err
 	}) == nil
 }
 
 // getEpochBucket returns bucket from db associated with given epoch
-func getEpochBucket(tx *bolt.Tx, epoch uint64) (*bolt.Bucket, error) {
-	epochBucket := tx.Bucket(epochsBucket).Bucket(common.EncodeUint64ToBytes(epoch))
+func getEpochBucket(tx *bolt.Tx, epoch uint64, chainID uint64) (*bolt.Bucket, error) {
+	epochBucket := tx.Bucket(epochsBucket).
+		Bucket(common.EncodeUint64ToBytes(chainID)).
+		Bucket(common.EncodeUint64ToBytes(epoch))
```

Comment:
```suggestion
	epochBucket := tx.Bucket(epochsBucket).
		Bucket(common.EncodeUint64ToBytes(chainID)).
		Bucket(common.EncodeUint64ToBytes(epoch))
```
Pay attention on order, it should be the same everywhere